string  object_name
int32[] bound_box
float32 confidence